test/extended/testdata/bindata.go

```diff
@@ -20136,7 +20149,8 @@ var _examplesQuickstartsDancerMysqlJson = []byte(`{
   },
   "message": "The following service(s) have been created in your project: ${NAME}, ${DATABASE_SERVICE_NAME}.\n\nFor more information about using this template, including OpenShift considerations, see https://github.com/openshift/dancer-ex/blob/master/README.md.",
   "labels": {
-    "template": "dancer-mysql-example"
+    "template": "dancer-mysql-example",
+    "app": "dancer-mysql-example"
```

Comment:
The `app` label should be parameterized so that when templates are used in conjunction to form a single application we can make the `app` labels match.